test/functionalTest/cases/3001_mqtt_alarms/mqtt_alarms_raise_and_release.test

```diff
@@ -238,8 +239,8 @@ Date: REGEX(.*)
 
 
 
-05. Get sub and see status:failed, timesSent:2 and lastFailure (but no lastSuccess)
-===================================================================================
+05. Get sub and see status:failed, timesSent:1, failsCounter:2 and lastFailure (but no lastSuccess)
+===================================================================================================
 HTTP/1.1 200 OK
 Content-Length: 404
```

Comment:
```suggestion
Content-Length: 421
```
This needs to be updated - The reason why tests are failling

Reply:
Good catch! Thanks!

Reply:
Fixed in ff98ebc